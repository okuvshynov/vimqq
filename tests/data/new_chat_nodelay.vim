let s:path = expand('<sfile>:p:h')
let s:lib = s:path . "/../libtest.vim"
execute "source " . s:lib

function! WriteAndQuit(t)
    :QQList

    let content = getline(1, '$')
    let expected = readfile(s:path . '/' . 'new_chat_nodelay.out')
    if VQQCompareChats(content, expected) == 0
        cquit 0
    else
        cquit 1
    endif
endfunction

:Q @mock hello
:Q -n @mock world!
call timer_start(1000, "WriteAndQuit")

