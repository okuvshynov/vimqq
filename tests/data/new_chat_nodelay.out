00:00>l=130
00:00 l=129
